### ISSUE TITLE
An enormous different accuracy caused by different optimizer

### ISSUE THREAD

**Issue by GuoHao150** [skorch-dev/skorch#468]
I'm trying to reproduce the results from skorch's official tutorial `https://github.com/skorch-dev/skorch/blob/master/notebooks/MNIST.ipynb` and at the part of `Convolutional Network` it works well until I changed the optimizer from `torch.optim.Adadelta` to `torch.optim.Adam` and then some weird thing happened that all the valid_acc returned from each epoch is 0.0987 which is almost zero and train_loss train_loss are both `nan`. When I changed the optimizer back to `torch.optim.Adadelta` the valid_acc reached 0.90 after seven epochs. I'm rookie at deep leaning but intuitively I think the difference is anormal I wonder if anyone here could give me an explanation. Thanks

**Comment by BenjaminBossan** [comment]
I don't think this it out of line. You should especially look at the learning rate, which needs to be vastly different for different optimizers -- for Adam, try to use a much smaller learning rate.

For a nice guide on training neural networks, have a look at [Karpathy's blog post](http://karpathy.github.io/2019/04/25/recipe/).

Please report back if you still find strange behavior, otherwise feel free to close the issue.

**Comment by GuoHao150** [comment]
My appreciation,  your reply solved my confusion
